src/content-tags-drawer/ContentTagsCollapsible.scss

```diff
@@ -0,0 +1,32 @@
+.taxonomy-tags-collapsible {
+  flex: 1;
+  border: none !important;
+
+  .collapsible-trigger {
+    border: none !important;
+  }
+}
+
+.tags-count-badge {
+  margin-top: 1rem;
```

Comment:
This is equivalent to `mt-3`

Reply:
Updated https://github.com/openedx/frontend-app-course-authoring/pull/654/commits/1c9d0f8c03cc3ab8a41a192426c8bc43792705c1